src/mscorlib/model.xml

```diff
@@ -1825,6 +1825,22 @@
       <Member Name="FailFast(System.String,System.Exception)"  />
       <Member Name="Exit(System.Int32)" />
     </Type>
+    <Type Name="Internal.Runtime.Augments.EnvironmentAugments">
```

Comment:
No.  If nothing else, corefx master still depends on it.  But even after that's changed, I'm not sure we can remove the existing Environment exports, in case we need to support an existing System.Runtime.Extensions.dll working with a newer System.Private.Corelib.dll... not sure if that's a supported scenario or not.